### PR TITLE
Misc fixes to get MDS working properly

### PR DESCRIPTION
**Pull request by ramereth**

### CODE REVIEW

**Comment by ramereth** [comment]
[osl-ceph cookbook](https://trello.com/c/nn6IXClg/636-osl-ceph-cookbook)

File: recipes/mds.rb [hunk 1]
```diff
@@ -23,19 +23,22 @@
...
-    action :create
-    not_if "test -d /var/lib/ceph/mds/#{cluster}-#{node['hostname']}"
-  end
+directory "/var/lib/ceph/mds/#{cluster}-#{node['hostname']}" do
+  owner node['ceph']['owner']
+  group node['ceph']['group']
+  mode node['ceph']['mode']
+  recursive true
+  action :create
+  not_if "test -d /var/lib/ceph/mds/#{cluster}-#{node['hostname']}"
```
Comment (doublej472) [inline recipes/mds.rb:32]: I think the `:create` action already tests something similar to this, is there a reason to have this explicitly specified?

**Review by doublej472 — CHANGES_REQUESTED** [review]

File: recipes/mds.rb (hunk 1 above)
Comment (ramereth) [reply recipes/mds.rb:32]: I just used the same code as before and removed it from the block.

**Review by doublej472 — APPROVED** [review]
Can't really test this cookbook, but the changes seem to be okay, so +1!

(We really need to fix testing on this cookbook)

**Review by doublej472 — APPROVED** [review]

**Pull request merged**